components/designSystem/SideMenu/SideMenuProfile/CoinBalance.tsx

```diff
@@ -0,0 +1,36 @@
+import {type ReactElement} from 'react';
+import {useBalance} from 'wagmi';
+import {useIsMounted} from '@react-hookz/web';
+import {useWeb3} from '@yearn-finance/web-lib/contexts/useWeb3';
+import {useChainID} from '@yearn-finance/web-lib/hooks/useChainID';
+import {getNetwork} from '@yearn-finance/web-lib/utils/wagmi/utils';
+import {Counter} from '@common/Counter';
+
+export function CoinBalance(): ReactElement {
+	const isMounted = useIsMounted();
```

Comment:
Haha feel free to ingore but this `isMounted: () => boolean` triggers me a bit.. It should be either boolean type or named like `getIsMounted`. If this continues triggering me I might add wrapper for this if you don't mind xd

Reply:
UPD: Thanks for adding one haha! Let's use `/hooks/useIsMounted.ts` instead of this one